map_gui/src/load.rs

```diff
@@ -327,6 +380,11 @@ mod wasm_loader {
         panel: Panel,
         started: Instant,
         url: String,
+
```

Comment:
I'm unhappy about copying all of the code to `RawFileLoader`, but I've still not managed to reconcile these two. All we need to do here is just return a raw `Vec<u8>` instead of plumbing through serde. https://github.com/a-b-street/abstreet/commit/657e56d7967fd84f9f56744e8eb3aedb9f596ae3 was an old attempt at this that failed.